core-bundle/src/Resources/contao/classes/Crawl.php

```diff
@@ -86,8 +86,16 @@ public function run()
 
 		$jobId = Input::get('jobId');
 		$queue = $factory->createLazyQueue();
-		$debugLogPath = sys_get_temp_dir() . '/contao-crawl/' . $jobId . '_log.csv';
-		$resultCache = sys_get_temp_dir() . '/contao-crawl/' . $jobId . '.result-cache';
+		$crawLogsDir = sys_get_temp_dir() . '/contao-crawl';
+
+		// Make sure the subdirectory exists so logs can be written
+		if (!is_dir($crawLogsDir))
+		{
+			mkdir($crawLogsDir);
```

Comment:
We should use `Filesystem::mkdir()` here. More robust implementation and better error handling.